flyway-core/src/main/java/org/flywaydb/core/internal/database/yugabytedb/YugabyteDBParser.java

```diff
@@ -0,0 +1,11 @@
+package org.flywaydb.core.internal.database.yugabytedb;
+
+import org.flywaydb.core.api.configuration.Configuration;
+import org.flywaydb.core.internal.database.postgresql.PostgreSQLParser;
+import org.flywaydb.core.internal.parser.ParsingContext;
+
+public class YugabyteDBParser extends PostgreSQLParser {
```

Comment:
Is this even required? What if we just use the Postgres parser? What about our custom ddl extensions and all? Won't they need some augmented parser for YB's specific ddls?

Reply:
Similar question for YugabyteDBSchema and YugabyteDBTable. Like why do we even have to extend when we are kind of not overriding anything?

Reply:
Yes. PostgreSQL implements a lot other methods from the respective base interfaces which we need not have to, if we simply extend these classes. In future, if we require to change any behaviour, we just need to override those methods.